docs/src/main/sphinx/sql/create-materialized-view.rst

```diff
@@ -0,0 +1,90 @@
+========================
+CREATE MATERIALIZED VIEW
+========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    CREATE [ OR REPLACE ] MATERIALIZED VIEW
+    [ IF NOT EXISTS ] view_name
+    [ COMMENT string ]
+    [ WITH properties ]
+    AS query
+
+Description
+-----------
+
+Create a new materialized view of a :doc:`select` query. The view is a physical
+manifestation of the query results at time of creation. The data is stored and
+can be referenced by future queries. Since the data is stored directly, these
+queries are typically faster than retrieving data from a view created with the
+same query.
+
+However the data in the materialized becomes out of sync with the underlying
+data in the source tables, when that data changes. The data in the materialized
+view is updated with the :doc:`refresh-materialized-view` statement.
+
+The optional ``OR REPLACE`` clause causes the materialized view to be replaced
+if it already exists rather than raising an error.
+
+The optional ``IF NOT EXISTS`` clause checks for availability of a materialized
+view of the same name and skips creation and execution of the query if a
+materialized view exists.
```

Comment:
Feels redundant, would just say "if one exists" or similar

Reply:
reworded and corrected in more detail